app/Http/Transformers/UsersTransformer.php

```diff
@@ -24,7 +24,7 @@ public function transformUser(User $user)
         $array = [
                 'id' => (int) $user->id,
                 'avatar' => e($user->present()->gravatar),
-                'name' => e($user->first_name).' '.e($user->last_name),
+                'name' => e($user->getFullNameAttribute()),
```

Comment:
I'm sure you already know this but because this is an accessor on the user model we can do `$user->full_name` instead of having to call the method.

Is calling the method an intentional pattern that I should start following?

Reply:
I'm not sure what you mean here?

Reply:
The `get*Attribute()` syntax tells Laravel to allow it to be [accessed](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor) like a "normal" property so you could do `$user->full_name`.

I'm 99% sure there is no benefit doing it that way but I normally don't see the full `get*Attribute()` call in code.